examples/Read/Read.ino

```diff
@@ -0,0 +1,38 @@
+/* Sweep
+ by BARRAGAN <http://barraganstudio.com>
+ This example code is in the public domain.
+
+ modified 18 Mar 2020
+ by S Jeeva
+ http://www.arduino.cc/en/Tutorial/Sweep
```

Comment:
```suggestion
```
This is no longer the Sweep example, so I don't think it's a good idea to link to that tutorial.